src/Zynga/Framework/PgData/V1/Test/ExampleFeature/Model/Sharded/ShardedInventoryFactory.hh

```diff
@@ -0,0 +1,19 @@
+<?hh //strict
+
+namespace Zynga\Framework\PgData\V1\Test\ExampleFeature\Model\Sharded;
+
+use Zynga\Framework\PgData\V1\Interfaces\PgModelInterface;
+use Zynga\Framework\PgData\V1\PgShardedModelFactory;
+use
+  Zynga\Framework\PgData\V1\Test\ExampleFeature\Model\Sharded\ShardedInventoryModel
+;
+use Zynga\Framework\Type\V1\UInt64Box;
+
+class ShardedInventoryFactory extends PgShardedModelFactory {
+
+  public function __construct(private UInt64Box $_shardKey) {}
```

Comment:
Does it make sense to define the constructor in PgShardedModelFactory with `private TypeInterface $_shardId;` that you can override here?